images/win/windows2016.json

```diff
@@ -254,6 +254,7 @@
                 "{{ template_dir }}/scripts/Installers/Install-IEWebDriver.ps1",
                 "{{ template_dir }}/scripts/Installers/Install-Apache.ps1",
                 "{{ template_dir }}/scripts/Installers/Install-Nginx.ps1"
+                "{{ template_dir }}/scripts/Installers/Install-swig.ps1"
```

Comment:
nitpick: Install-Swig.ps1

Reply:
Thanks. File name is Install-swig.ps1, not a Install-Swig.ps1.

Reply:
To be consistence with other scripts name. I think better to rename Install-swig.ps1 -> Install-Swig.ps1